scripts/fetch_whitelisted_tags.py

```diff
@@ -119,20 +119,27 @@ def construct_versions_map(remotes, vehicles):
             s = ref.split('/', 3)
 
             # skip if tag name does not start with custom-build/
-            if s[0] != 'custom-build':
+            if len(s) <= 1 or s[0] != 'custom-build':
```

Comment:
naming these will make things clearer.

So `(x, y, y) = s`

(but useful names in place of x etc